gbpservice/neutron/nsf/configurator/api/setup.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+try:
```

Comment:
No. This also gets created  as a part of pecan create [project_name] command. But I assumed that we are not going to run that command and directly paste our directory so I added this with required modification.